### PR TITLE
Creates ProdMeter and adds relevent elements for CEC fields to Object.

### PR DESCRIPTION
**Pull request by kevinkirkl3y**
Creates ProdMeter and updates WhatsNew documents.

### CODE REVIEW

File: Master-OB-OpenAPI.json [hunk 1]
```diff
@@ -10341,6 +10341,65 @@
...
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/TaxonomyElementNumber"
+          },
+          {
+            "type": "object",
+            "description": "Energy measurement accuracy, as defined by ANSI C12.20. The value is percent over or under the exact measurement. ",
+            "x-ob-item-type": "percentItemType",
+            "x-ob-item-type-group": "",
+            "x-ob-usage-tips": "In 2022 if it is marked for Performance Based Incentive (PBI) then enter 2% or less. ",
```
Comment (cwhanse) [inline Master-OB-OpenAPI.json:10379]:
```suggestion
            "x-ob-usage-tips": "In 2022 if the meter is marked for Performance Based Incentive (PBI) then enter 2% or less. ",
```

File: Master-OB-OpenAPI.json [hunk 2]
```diff
@@ -10341,6 +10341,65 @@
...
+      },
+      "AccuracyClassANSI": {
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/TaxonomyElementNumber"
+          },
+          {
+            "type": "object",
+            "description": "Energy measurement accuracy, as defined by ANSI C12.20. The value is percent over or under the exact measurement. ",
+            "x-ob-item-type": "percentItemType",
```
Comment (reepoi) [inline Master-OB-OpenAPI.json:10377]:
```suggestion
            "x-ob-item-type": "PercentItemType",
```

File: Master-OB-OpenAPI.json [hunk 3]
```diff
@@ -10341,6 +10341,65 @@
...
+      },
+      "DisplayDescription": {
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/TaxonomyElementString"
+          },
+          {
+            "type": "object",
+            "description": "Type of display on a device. ",
+            "x-ob-item-type": "stdi:stringItemType",
```
[the remark this answers is not in the document]
Comment (reepoi) [reply Master-OB-OpenAPI.json:10394]: and `stringItemType` should be `StringItemType`.

File: Master-OB-OpenAPI.json [hunk 4]
```diff
@@ -10341,6 +10341,65 @@
...
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/TaxonomyElementNumber"
+          },
+          {
+            "type": "object",
+            "description": "Energy measurement accuracy, as defined by ANSI C12.20. The value is percent over or under the exact measurement. ",
+            "x-ob-item-type": "PercentItemType",
+            "x-ob-item-type-group": "",
+            "x-ob-usage-tips": "In 2022 if it the meter is marked for Performance Based Incentive (PBI) then enter 2% or less. ",
```
Comment (cwhanse) [inline Master-OB-OpenAPI.json:10379]:
```suggestion
            "x-ob-usage-tips": "In 2022 if the meter is marked for Performance Based Incentive (PBI) then enter 2% or less. ",
```

**Pull request merged**